core/src/edu/asu/c3simulator/screens/MarketingScreen.java

```diff
@@ -0,0 +1,368 @@
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+import edu.asu.c3simulator.util.Product;
+import edu.asu.c3simulator.widgets.NavigationPanel;
+
+/**
+ * Display a list of all products created by the business and give the user the ability to
+ * either market one of them or the company as a whole. The user is also able to choose
+ * from different types of marketing campaigns. This is the first step of two-step wizard.
+ * 
+ * @author Alyahya, Mohammed
+ */
+public class MarketingScreen implements Screen
+{
+	
+	private class productIconListener extends ClickListener
+	{
+		private Label associatedLabel;
+		private Product associatedProduct;
+		
+		/**
+		 * @param associatedLabel
+		 *            A label which will change colors when the icon is clicked.
+		 * @param associatedProduct
+		 *            The product that is represented by the object that is being listened
+		 *            to.
+		 */
+		public productIconListener(Label associatedLabel, Product associatedProduct)
+		{
+			this.associatedLabel = associatedLabel;
+			this.associatedProduct = associatedProduct;
+		}
+		
+		@Override
+		public void clicked(InputEvent event, float x, float y)
+		{
+			companyPromotionButton.setColor(1.0f, 1.0f, 1.0f, 1.0f);
+			associatedLabel.setColor(0.0f, 1.0f, 0.0f, 1.0f);
+			if (selectedProductLabel != null && selectedProductLabel != associatedLabel)
+				selectedProductLabel.setColor(1.0f, 1.0f, 1.0f, 1.0f);
+			
+			selectedProduct = associatedProduct;
+			selectedProductLabel = associatedLabel;
+			productChosen = true;
+		}
+	}
+	
+	private class marketingIconListener extends ClickListener
+	{
+		private Label associatedLabel;
+		private String marketingType;
+		
+		/**
+		 * @param associatedLabel
+		 *            A label which will change colors when the icon is clicked
+		 * @param marketingType
+		 *            The marketing type that the icon is representing.
+		 */
+		public marketingIconListener(Label associatedLabel, String marketingType)
+		{
+			this.associatedLabel = associatedLabel;
+			this.marketingType = marketingType;
+		}
+		
+		@Override
+		public void clicked(InputEvent event, float x, float y)
+		{
+			associatedLabel.setColor(0.0f, 0.0f, 1.0f, 1.0f);
+			if (selectedMarketingLabel != null
+					&& selectedMarketingLabel != associatedLabel)
+				selectedMarketingLabel.setColor(1.0f, 1.0f, 1.0f, 1.0f);
+			
+			selectedMarketingMethod = marketingType;
+			selectedMarketingLabel = associatedLabel;
+			marketingMethodChosen = true;
+		}
+	}
+	
+	private Game game;
+	private Stage stage;
+	private Skin skin;
+	private Product selectedProduct;
+	private Label selectedProductLabel, selectedMarketingLabel;
+	private TextButton companyPromotionButton;
+	private String selectedMarketingMethod = "";
+	private boolean promoteCompany = false, productChosen = false,
+			marketingMethodChosen = false;
```

Comment:
New lines for these.
Also, the default is false, but if you wanted to clarify, the constructor would be a better place to do so.